### PR TITLE
DOC-6833 MLOps update

### PR DESCRIPTION
**Pull request by paperclypse**
I've attempted to clean up this doc a little bit.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx [hunk 1]
```diff
@@ -317,114 +397,34 @@ You can get data from any of the following sources:
     </tr>
         <tr>
       <td>
-        3. Configure Algorithmia Insights for New Relic
+        3. Set up alerts notifications
       </td>
       <td>
-        Use [Algorithmia's docs] to configure Algorithmia Insights for New Relic.
+        Once you've created some dashboards, you can get alerted on your data.  To create a NRQL alerts condition from a chart, click the chart menu <Icon name="fe-more-horizontal"/>, then click **Create alert condition**. Once you've named and customized your condition, you can add it to an existing policy or create a new one.
```
Comment (urbiz-nr) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx:403]: Super minor, but maybe add a comma before "create a new one"

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx [hunk 2]
```diff
@@ -7,42 +7,42 @@ tags:
...
 
-MLOps allows teams to alleviate these challenges by providing a technological backbone for managing the machine learning lifecycle through automation and scalability. This allows for a seamless collaboration between data teams and DevOp teams ultimately leading to innovative solutions through the creation of sophisctated ML models, continued development and testing, and consistent operational and monitoring of the software. 
+## The Algorithmia integration [#algorithmia]
 
+Algorithmia Insights provides tools for deploying your machine-learning models into production. By integrating, Algorithmia's integration with New Relic, you'll be able to instrument, analyze, troubleshoot, and optimize your machine-learning performance across your entire system. By rigorously observing your capabilities, you'll be able to react quickly to changes in the model's input or output and the relationship between the two.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:18]: The beginning of the second sentence is a little odd. How about "Integrate Algorithmia with New Relic to instrument, analyze..."

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx [hunk 3]
```diff
@@ -7,42 +7,42 @@ tags:
...
 
-## Integration with Algorithmia
+Send your model performance metrics from Algorithmia Insights to New Relic and you'll have real-time monitoring for your algorithms. You'll explore your metrics data with user-friendly charts and learn the state of your algorithms at a glance for faster and more efficient troubleshooting.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:20]: I would avoid future tense.

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx [hunk 4]
```diff
@@ -7,42 +7,42 @@ tags:
...
+## Integrate Algorithmia with New Relic [#integrate-algorithmia] 
 
-Since New Relic’s cloud-based observability infrastructure already helps instrument, analyze, troubleshoot, and optimize your entire software stack, Algorithmia is the perfect solution to collect model performance data and connect with New Relic’s monitoring tools - allowing for a smooth integration for business and operational management. 
-
-By sending your model performance metrics from Algorithmia Insights to New Relic and having real-time monitoring for your algorithms, you can explore your Metrics data with user-friendly charts, and quickly learn the state of your algorithms for faster and more efficient troubleshooting. 
+First, Algorithmia uses a Kafka topic to stream Insights. Your machine-learning algorithm's performance metrics. Then, the New Relic connector (another algorithm) transforms the Kafka topic into a metrics data payload for a specific New Relic account.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:24]: I don't understand this paragraph. What is the second sentence? There is no predicate...

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx [hunk 5]
```diff
@@ -7,42 +7,42 @@ tags:
...
+    By integrating Incident Intelligence with your Algorithmia machine-learning models, you can monitor your machine learning model performance.
 
-Use the following instructions to set up this integration:
+  Start monitoring your Algorithmia event flows with New Relic.
 
-### Step 1: Get your API key
+1. **Get your API key:** From [one.newrelic.com](https://one.newrelic.com) the account menu, click **API keys** and then create a user key for your account with a meaningful name. Make note of this name for later. For more on API keys, [see our docs](/docs/apis/get-started/intro-apis/new-relic-api-keys/).
```
Comment (urbiz-nr) [inline src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:38]: "From one.newrelic.com, go to the account menu, click..."

**Comment by urbiz-nr** [comment]
Looks really good, @paperclypse! I've left a few comments!

**Review by urbiz-nr — APPROVED** [review]

**Review by rhetoric101 — APPROVED** [review]

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:20]: This one is a weird amalgamation of marketing and docs. I'll do my best!

File: src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx:24]: Ha! All the signs of a distracted writer :) I fixed this.

**Pull request merged**